packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx

```diff
@@ -261,107 +267,110 @@ export const OAuthLoginPage = () => {
           txParams={txParams}
         />
       )}
-      {isLoggedIn ? (
-        <div className={styles.userInfoContainer}>
-          <h3 className={styles.infoSectionTitle}>{messages.signedInAs}</h3>
-          <div className={styles.tile}>
-            <ProfileInfo
-              displayNameClassName={styles.userInfoDisplayName}
-              handleClassName={styles.userInfoHandle}
-              centered={false}
-              imgClassName={styles.profileImg}
-              className={styles.userInfo}
-              user={account}
-            />
-          </div>
-          <div className={styles.signOutButtonContainer}>
-            <button className={styles.linkButton} onClick={handleSignOut}>
-              {messages.signOut}
-            </button>
-          </div>
-          <CTAButton
-            isSubmitting={isSubmitting}
-            text={
-              userAlreadyWriteAuthorized
-                ? messages.continueButton
-                : messages.authorizeButton
-            }
-            onClick={handleAlreadySignedInAuthorizeSubmit}
-          />
-        </div>
-      ) : (
-        <div className={styles.signInFormContainer}>
-          <form onSubmit={handleSignInFormSubmit}>
-            <Input
-              placeholder='Email'
-              size='medium'
-              type='email'
-              name='email'
-              id='email-input'
-              required
-              autoComplete='username'
-              value={emailInput}
-              onChange={handleEmailInputChange}
-            />
-            <Input
-              className={styles.passwordInput}
-              placeholder='Password'
-              size='medium'
-              name='password'
-              id='password-input'
-              required
-              autoComplete='current-password'
-              value={passwordInput}
-              type='password'
-              onChange={setPasswordInput}
-            />
-            {signInError == null ? null : (
-              <div className={styles.credentialsErrorContainer}>
-                <IconValidationX
-                  width={14}
-                  height={14}
-                  className={styles.credentialsErrorIcon}
-                />
-                <span className={styles.errorText}>{signInError}</span>
-              </div>
-            )}
-            {showOtpInput ? (
-              <Input
-                placeholder='Verification Code'
-                size='medium'
-                name='otp'
-                value={otpInput}
-                characterLimit={6}
-                type='number'
-                variant={'normal'}
-                onChange={setOtpInput}
-                className={cn(styles.otpInput)}
+      <div className={styles.formArea}>
```

Comment:
This looks crazy but I just added a div wrapper

Reply:
didnt know you were crazy like that
![](https://media.giphy.com/media/lo8YzDnHh07rpxqw45/giphy.gif)